packages/core/src/infrastructure/database/migrations/2024-03-31T21:37:51Z-share-email.ts

```diff
@@ -0,0 +1,46 @@
+import { Kysely } from 'kysely';
+
+export async function up(db: Kysely<any>) {
+  await db.schema
+    .alterTable('students')
+    .addColumn('allow_email_share', 'boolean')
```

Comment:
Let's combine all of the previous statements into one!

```
await db.schema
  .alterTable('students')
  .addColumn('allow_email_share', 'boolean', (column) => {
    return column.notNull().defaultTo(true);
  })
```

Sorry for the confusion, I know the other 2 migration files probably didn't make it clear that we could do this!